core/src/button/button.tsx

```diff
@@ -267,10 +274,11 @@ const buttonRender = (
 	return props.href ? (
 		<a
 			{...common}
+			onClick={props.onClick}
 			href={props.href}
-			target={props.target}
 			download={props.download}
-			rel="noopener noreferrer"
+			target={props.target}
+			rel={props.rel || "noopener noreferrer"}
```

Comment:
hm we should not default to "noopener..." anyway :( but removing that would break version... so please add this note to #230